src/main/java/seedu/address/model/room/RoomList.java

```diff
@@ -67,7 +76,19 @@ public void addRooms(int numOfRooms) {
         this.numOfRooms = numOfRooms;
         addRooms();
     }
-
+    /**
+     * Sets the elements of {@code roomObservableList}.
+     *
+     * @param roomList PriorityQueue containing all the rooms.
+     */
+    private void convertPriorityQueue(PriorityQueue<Room> roomList) {
```

Comment:
If we have to convert to priority queue, do you guys think if it's better to use array list directly instead of priority queue for room list?

Reply:
i think instead of arraylist, it might be better to use ObservableList directly. ObservableList is what is passed into ListView (UI component) and allows the UI to be updated whenever there is a change in ObservableList.

Reply:
Right, that's true!